pkg/acquisition/modules/file/file.go

```diff
@@ -330,7 +330,21 @@ func (f *FileSource) StreamingAcquisition(out chan types.Event, t *tomb.Tomb) er
 			continue
 		}
 
-		tail, err := tail.TailFile(file, tail.Config{ReOpen: true, Follow: true, Poll: f.config.PollWithoutInotify, Location: &tail.SeekInfo{Offset: 0, Whence: io.SeekEnd}, Logger: log.NewEntry(log.StandardLogger())})
+		inotifyPoll := true
+		if f.config.PollWithoutInotify != nil {
+			inotifyPoll = *f.config.PollWithoutInotify
+		} else {
+			networkFS, fsType, err := types.IsNetworkFS(file)
+			if err != nil {
+				f.logger.Warningf("Could not get fs type for %s : %s", file, err)
+			}
+			f.logger.Debugf("fs for %s is network: %t (%s)", file, networkFS, fsType)
+			if networkFS {
+				inotifyPoll = false
```

Comment:
Is this worth explicit logging ? "Disabling inotify poll on %s as it is a network share. You can manually set inotify to false to make this message disappear, or to true to enforce inotify poll"